src/unix/linux-inotify.c

```diff
@@ -108,6 +108,40 @@ static int init_inotify(uv_loop_t* loop) {
 }
 
 
+int uv__inotify_fork(uv_loop_t* loop, void* old_watchers) {
+  /* open the inotify_fd, and re-arm all the inotify watchers */
+  int err;
+  struct watcher_list* tmp_watcher_list;
+  struct watcher_list* watcher_list;
+  QUEUE* q;
+  uv_fs_event_t* handle;
+  char* path;
+
+  if (old_watchers) {
+    /* We must restore the old watcher list to be able to close items
+     *  out of it.
+     */
+    loop->inotify_watchers = old_watchers;
+
+    err = init_inotify(loop);
```

Comment:
why do we need to do this here if the first call to `uv_fs_event_start` will already do it?